UAV1:
  obs_vars: [obs_fire]

  obs_fire:
    values: [video, thermal, none, unknown]

    u:
      scout1: [0.045, 0.045, 0.895, 0.015]
      scout2: [0.045, 0.045, 0.895, 0.015]
      scout3: [0.045, 0.045, 0.895, 0.015]
      scout4: [0.045, 0.045, 0.895, 0.015]
      scout5: [0.045, 0.045, 0.895, 0.015]
      scout6: [0.045, 0.045, 0.895, 0.015]
      scout7: [0.045, 0.045, 0.895, 0.015]
      scout8: [0.045, 0.045, 0.895, 0.015]
      scout9: [0.045, 0.045, 0.895, 0.015]
      scout10: [0.045, 0.045, 0.895, 0.015]
      end: [0.25, 0.25, 0.25, 0.25]

    f1:
      scout1: [0.45, 0.45, 0.09, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]
    
    f2:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.45, 0.45, 0.09, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]
    
    f3:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.45, 0.45, 0.09, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]
    
    f4:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.45, 0.45, 0.09, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f5:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.45, 0.45, 0.09, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25,]

    f6:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.45, 0.45, 0.09, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f7:
      scout1: [0.45, 0.45, 0.09, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.45, 0.45, 0.09, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f8:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.45, 0.45, 0.09, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f9:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.45, 0.45, 0.09, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f10:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.45, 0.45, 0.09, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

UAV2:
  obs_vars: [obs_fire]

  obs_fire:
    values: [video, thermal, none, unknown]

    u:
      scout1: [0.045, 0.045, 0.895, 0.015]
      scout2: [0.045, 0.045, 0.895, 0.015]
      scout3: [0.045, 0.045, 0.895, 0.015]
      scout4: [0.045, 0.045, 0.895, 0.015]
      scout5: [0.045, 0.045, 0.895, 0.015]
      scout6: [0.045, 0.045, 0.895, 0.015]
      scout7: [0.045, 0.045, 0.895, 0.015]
      scout8: [0.045, 0.045, 0.895, 0.015]
      scout9: [0.045, 0.045, 0.895, 0.015]
      scout10: [0.045, 0.045, 0.895, 0.015]
      end: [0.25, 0.25, 0.25, 0.25]

    f1:
      scout1: [0.45, 0.45, 0.09, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]
    
    f2:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.45, 0.45, 0.09, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]
    
    f3:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.45, 0.45, 0.09, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]
    
    f4:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.45, 0.45, 0.09, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f5:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.45, 0.45, 0.09, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25,]

    f6:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.45, 0.45, 0.09, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f7:
      scout1: [0.45, 0.45, 0.09, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.45, 0.45, 0.09, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f8:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.45, 0.45, 0.09, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f9:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.45, 0.45, 0.09, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f10:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.45, 0.45, 0.09, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

UAV3:
  obs_vars: [obs_fire]

  obs_fire:
    values: [video, thermal, none, unknown]

    u:
      scout1: [0.045, 0.045, 0.895, 0.015]
      scout2: [0.045, 0.045, 0.895, 0.015]
      scout3: [0.045, 0.045, 0.895, 0.015]
      scout4: [0.045, 0.045, 0.895, 0.015]
      scout5: [0.045, 0.045, 0.895, 0.015]
      scout6: [0.045, 0.045, 0.895, 0.015]
      scout7: [0.045, 0.045, 0.895, 0.015]
      scout8: [0.045, 0.045, 0.895, 0.015]
      scout9: [0.045, 0.045, 0.895, 0.015]
      scout10: [0.045, 0.045, 0.895, 0.015]
      end: [0.25, 0.25, 0.25, 0.25]

    f1:
      scout1: [0.45, 0.45, 0.09, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]
    
    f2:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.45, 0.45, 0.09, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]
    
    f3:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.45, 0.45, 0.09, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]
    
    f4:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.45, 0.45, 0.09, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f5:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.45, 0.45, 0.09, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25,]

    f6:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.45, 0.45, 0.09, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f7:
      scout1: [0.45, 0.45, 0.09, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.45, 0.45, 0.09, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f8:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.45, 0.45, 0.09, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f9:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.45, 0.45, 0.09, 0.01]
      scout10: [0.045, 0.045, 0.9, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]

    f10:
      scout1: [0.045, 0.045, 0.9, 0.01]
      scout2: [0.045, 0.045, 0.9, 0.01]
      scout3: [0.045, 0.045, 0.9, 0.01]
      scout4: [0.045, 0.045, 0.9, 0.01]
      scout5: [0.045, 0.045, 0.9, 0.01]
      scout6: [0.045, 0.045, 0.9, 0.01]
      scout7: [0.045, 0.045, 0.9, 0.01]
      scout8: [0.045, 0.045, 0.9, 0.01]
      scout9: [0.045, 0.045, 0.9, 0.01]
      scout10: [0.45, 0.45, 0.09, 0.01]
      end: [0.25, 0.25, 0.25, 0.25]